### DISCUSSION TITLE
file

### DISCUSSION THREAD

**Discussion by yoyoriri** [aiko-chan-ai/DiscordBotClient#107] [Q&A] [answered]
why can't we send more than 25 MB ?

**Comment by Wissididom** [comment]
Probably because of this:
> A file upload size limit applies to all files in a request (rather than each individual file). While the limit depends on the [Boost Tier](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-FAQ-#h_419c3bd5-addd-4989-b7cf-c7957ef92583) of a guild, it is 25 MiB by default.

(Source: https://discord.com/developers/docs/reference#uploading-files)

**Reply by yoyoriri** [reply]
yes but since we have nitro it's normally 50 MB?

**Reply by Wissididom** [reply, marked as answer]
Yes, but you are using a bot which doesn't have Nitro (afaik cannot even have Nitro).
